SPECS/abseil-cpp/abseil-cpp.spec

```diff
@@ -34,45 +44,46 @@ Abseil is not meant to be a competitor to the standard library; we've just
 found that many of these utilities serve a purpose within our code base,
 and we now want to provide those resources to the C++ community as a whole.
 
+%package testing
+Summary:        Libraries needed for running tests on the installed %{name}
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+
+%description testing
+%{summary}.
+
 %package devel
 Summary:        Development files for %{name}
-Requires:       %{name} = %{version}-%{release}
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+Requires:       %{name}%{?_isa} = %{version}-%{release}
 
 %description devel
 Development headers for %{name}
 
 %prep
-%autosetup -p1
+%autosetup -p1 -S gendiff
 
 %build
-mkdir build
-pushd build
 %cmake \
-  -DABSL_PROPAGATE_CXX_STD=ON \
-  -DCMAKE_BUILD_TYPE=RelWithDebInfo \
-%if 0%{?with_check}
-  -DABSL_BUILD_TESTING=ON \
-  -DABSL_FIND_GOOGLETEST=ON \
-  -DABSL_USE_EXTERNAL_GOOGLETEST=ON \
-  -DBUILD_TESTING=ON \
-%else
-  -DBUILD_TESTING=OFF \
-%endif
-  ..
-%make_build
+  -GNinja \
+  -DABSL_USE_EXTERNAL_GOOGLETEST:BOOL=ON \
+  -DABSL_FIND_GOOGLETEST:BOOL=ON \
+  -DABSL_ENABLE_INSTALL:BOOL=ON \
+  -DABSL_BUILD_TESTING:BOOL=ON \
+  -DABSL_BUILD_TEST_HELPERS:BOOL=ON \
+  -DCMAKE_BUILD_TYPE:STRING=None \
+  -DCMAKE_CXX_STANDARD:STRING=17
+%cmake_build
 
 %install
-pushd build
-%make_install
+%cmake_install
 
 %check
-pushd build
-ctest --output-on-failure
+%ctest --output-on-failure
 
 %files
 %license LICENSE
 %doc FAQ.md README.md UPGRADES.md
-%{_libdir}/libabsl_*.so.2308.*
+%{_libdir}/libabsl_*.so.2401.*
```

Comment:
Can you use a variable at the top to define this number? (based on lib_version ). It will make future upgrades easier